migrations/2_deploy_organization.js

```diff
@@ -42,12 +42,12 @@ module.exports = async (deployer) => {
     preBoostedVoteRequiredPercentage: 50,
     preBoostedVotePeriodLimit: 5184000, // 2 months
     boostedVotePeriodLimit: 604800, // 1 week
-    thresholdConstA: 2,
+    thresholdConstA: web3.toWei(2),
```

Comment:
I suggest to define GP default values separately and import it here,

Reply:
See https://github.com/daostack/arc.js/issues/171

Reply:
why not including that on this pr ?

Reply:
> why not including that on this pr ?

See response above.